ci/stack-orchestration/manual-stacks/txma/template.yml

```diff
@@ -0,0 +1,203 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: SQS queues for audit events
+
+Parameters:
+  Environment:
+    Description: 'The environment type'
+    Type: 'String'
+    AllowedValues:
+      - 'dev'
+      - 'build'
+      - 'staging'
+      - 'integration'
+      - 'production'
+    ConstraintDescription: must be dev, build, staging, integration or production
+
+Mappings:
+  TxMAAccountARN:
+    Environment:
+      dev: 'unused'
+      build: 'unused'
+      staging: 'arn:aws:iam::178023842775:root'
+      integration: 'arn:aws:iam::729485541398:root'
+      production: 'arn:aws:iam::451773080033:root'
+
+Conditions:
+  TxMAIntegrationEnabled:
+    Fn::Or:
+      - Fn::Equals: [!Ref Environment, "staging"]
+      - Fn::Equals: [!Ref Environment, "integration"]
+      - Fn::Equals: [!Ref Environment, "production"]
+
+Resources:
+  TxMASQSProducerAuditEventQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      MessageRetentionPeriod: 1209600 # 14 days in seconds
+      QueueName: !Sub '${AWS::StackName}-AuditEventQueue'
+      KmsMasterKeyId:
+        Fn::If:
+          - TxMAIntegrationEnabled
+          - !Ref TxMASQSProducerAuditEventQueueEncryptionKeyAlias
+          - !Ref AWS::NoValue
```

Comment:
Why this distinction?

Reply:
If `TxMAIntegrationEnabled = false` then `TxMASQSProducerAuditEventQueueEncryptionKeyAlias` isn't provisioned so can't be assigned to `KmsMasterKeyId`. Is that what you meant?

Reply:
Now conditionally provisioning only resources related to decryption of events by TxMA